_data/meltano/utilities/dagster/quantile-development.yml

```
@@ -0,0 +1,51 @@
+name: dagster
+label: Dagster
+domain_url: https://dagster.io/
+namespace: dagster
+pip_url: dagster-ext
+ext_repo: https://github.com/quantile-development/dagster-ext
+maintenance_status: development
+repo: https://github.com/dagster-io/dagster
+executable: dagster_invoker
+logo_url: /assets/logos/utilities/dagster.png
+variant: quantile-development
+commands:
+  describe:
+    executable: dagster_extension
+    args: describe
+    description: Describe the Dagster Extension.
+  init:
```

```suggestion
  initialize:
```

I think I'd messed up some of my commits previous. Let's keep this as `initialize`. I'd prefer we not have two commands that do the same thing, so dropping `init` makes sense and is consistent for now.

---

Alright, makes sense. I will remove `init`